Backend/nix/services/nammayatri.nix

```diff
@@ -76,10 +83,12 @@ in
           (name: {
             inherit name;
             value = {
-              imports = [ (haskellProcessFor name) ];
-              log_location = "${name}.log";
-              working_dir = "Backend";
+              imports = [
+                common
+                (haskellProcessFor name)
+              ];
               depends_on."nammayatri-init".condition = "process_completed_successfully";
+              shutdown.signal = 9;
```

Comment:
@srid this will ensure that the process-compose window doesn’t hang. Turns out the haskell apps run via `cabal run` doesn’t accept `SIGTERM` (not entirely sure via), sending SIGKILL is a little dangerous but it works for now.

Reply:
Let's add that as a code comment then, so people who read this line in the future immediately knows why it exists.

Reply:
I have added a comment for `SIGKILL`